recipes/ozo/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import check_min_cppstd, Version
+import os
+import glob
+
+
+class OzoConan(ConanFile):
+    name = "ozo"
+    description = "C++ header-only library for asynchronous access to PostgreSQL databases using ASIO"
+    topics = ("ozo", "yandex", "postgres", "postgresql", "cpp17", "database", "db", "asio")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yandex/ozo"
+    license = "PostgreSQL"
+
+    settings = "os", "arch", "compiler", "build_type"
+    requires = ("boost/1.75.0", "resource_pool/cci.20210322", "libpq/13.2")
```

Comment:
Theres a newer version of boost if its supported
```suggestion
    requires = ("boost/1.76.0", "resource_pool/cci.20210322", "libpq/13.2")
```